cluster/src/test/java/com/hartwig/pipeline/tertiary/cuppa/CuppaTest.java

```diff
@@ -1,24 +1,32 @@
 package com.hartwig.pipeline.tertiary.cuppa;
 
 import static com.hartwig.pipeline.Arguments.testDefaultsBuilder;
+import static com.hartwig.pipeline.testsupport.TestInputs.SOMATIC_BUCKET;
 import static com.hartwig.pipeline.testsupport.TestInputs.linxOutput;
 import static com.hartwig.pipeline.testsupport.TestInputs.purpleOutput;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 import java.util.List;
 
+import com.hartwig.pipeline.ResultsDirectory;
 import com.hartwig.pipeline.datatypes.DataType;
 import com.hartwig.pipeline.metadata.AddDatatype;
 import com.hartwig.pipeline.metadata.ArchivePath;
 import com.hartwig.pipeline.metadata.SomaticRunMetadata;
 import com.hartwig.pipeline.report.Folder;
 import com.hartwig.pipeline.stages.Stage;
+import com.hartwig.pipeline.storage.GoogleStorageLocation;
 import com.hartwig.pipeline.tertiary.TertiaryStageTest;
 import com.hartwig.pipeline.testsupport.TestInputs;
 
 public class CuppaTest extends TertiaryStageTest<CuppaOutput> {
 
+    private static final String TUMOR_CUPPA_CONCLUSION_TXT = "tumor.cuppa.conclusion.txt";
+    private static final String TUMOR_CUPPA_CHART_PNG = "tumor.cuppa.chart.png";
+    private static final String TUMOR_CUP_DATA_CSV = "tumor.cup.data.csv";
+    private static final String TUMOR_CUP_REPORT_SUMMARY_PNG = "tumor.cup.report.summary.png";
```

Comment:
Same as the implementation: inconsistent naming with the `CUP_`. Here the names match the constants though, but I guess still not the datatypes.